### PR TITLE
LPS-153133 StructuredContentAssetPriority#StructuredContentDraftContainsSetPriorityValue

### PR DESCRIPTION
**Pull request by mateusralv**
Background
Create automation tests for StructuredContentAssetPriority

Test Plan
**Given** a content structure with a content-field of dataType "string" and label "content" and name "content" created in the portal
**When** I create a structuredContent draft with Headless Admin Content POST "/v1.0/sites/{siteId}/structured-contents/draft" with a priority field set (as in the json example attached)
**Then** the response body includes the set priority

JIRA Ticket:
https://issues.liferay.com/browse/LPS-153133

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro [hunk 1]
```diff
@@ -89,7 +89,7 @@ definition {
 	}
 
 	macro _createStructuredContentDraft {
-		Variables.assertDefined(parameterList = "${data},${label},${name},${ddmStructureId},${title}");
+		Variables.assertDefined(parameterList = "${data},${label},${name},${ddmStructureId},${priority},${title}");
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro:93]: the `priority` is an optional value, if we make it `Variables.assertDefined` then we will have to add it everytime we want to create the StructuredContent

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro [hunk 2]
```diff
@@ -158,11 +159,16 @@ definition {
 	}
 
 	macro createStructuredContentDraft {
+		if (!(isSet(priority))) {
+			var priority = "";
+		}
+
 		var response = HeadlessWebcontentAPI._createStructuredContentDraft(
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro:166]: oh, okay I see what you did there :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase [hunk 3]
```diff
@@ -55,4 +55,27 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test StructuredContentDraftContainsSetPriorityValue {
+		property portal.acceptance = "true";
+
+		task ("When create a structuredContent draft with Headless Admin Content POST '/v1.0/sites/{siteId}/structured-contents/draft' with a priority field set (as in the json example attached)") {
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase:63]: I think we can shorten the step a bit here, for example, to `When in Headless Admin Content I create a structuredContent draft with with a priority field set`

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro (hunk 1 above)
Comment (magjed4289) [reply portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro:93]: nvm, ignore the comment :)

**Review by magjed4289 — CHANGES_REQUESTED** [review]

**Review by magjed4289 — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:test:frontend

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**